python/notebooks/06-memory-and-embeddings.ipynb

```diff
@@ -390,9 +515,126 @@
       "id": "59294dac",
       "metadata": {},
       "source": [
-        "Now you might be wondering what happens if you have so much data that it doesn't fit into your RAM? That's where you want to make use of an external Vector Database made specifically for storing and retrieving embeddings.\n",
+        "Now you might be wondering what happens if you have so much data that it doesn't fit into your RAM? That's where you want to make use of an external Vector Database made specifically for storing and retrieving embeddings. Fortunately, semantic kernel makes this easy thanks to an extensive list of available connectors. In the following section, we will connect to an existing Azure AI Search service that we will use as an external Vector Database to store and retrieve embeddings.\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 13,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "from semantic_kernel.connectors.memory.azure_cognitive_search import AzureCognitiveSearchMemoryStore\n",
         "\n",
-        "Stay tuned for that!"
+        "azure_ai_search_api_key, azure_ai_search_url = sk.azure_aisearch_settings_from_dot_env()\n",
+        "kernel.register_memory_store(memory_store=AzureCognitiveSearchMemoryStore(1536,search_endpoint=azure_ai_search_url,admin_key=azure_ai_search_api_key))\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "The implementation of Semantic Kernel allows to easily swap memory store for another. Here, we will re-use the functions we initially created for `VolatileMemoryStore` with our new external Vector Store leveraging Azure AI Search"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 14,
+      "metadata": {},
+      "outputs": [
+        {
+          "name": "stderr",
+          "output_type": "stream",
+          "text": [
+            "hnsw_parameters is not a known attribute of class <class 'azure.search.documents.indexes._generated.models._models_py3.HnswVectorSearchAlgorithmConfiguration'> and will be ignored\n"
+          ]
+        }
+      ],
+      "source": [
+        "await populate_memory(kernel)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "We can see that our function created an \"About Me\" index and that our five pieces of infomation have been indexed (note that it can take a few minutes for the UI to reflect the document count and storage size)."
+      ]
+    },
+    {
+      "attachments": {
+        "image.png": {
```

Comment:
I am having a difficult time reading the text in the images. Perhaps we could crop the image and show the most relevant part? This goes for the second image as well.

Reply:
Hi Evan, I hope this is addressed to your satisfaction in commit 957f3d7d52d05075b1c524f32c5e037713dc73ae